csharp-basic-topics/ValueAndReferenceTypes/ValueAndReferenceTypes/ReferenceTypeSample.cs

```diff
@@ -0,0 +1,26 @@
+﻿namespace ValueAndReferenceTypes
+{
+    public class ReferenceTypeRectangle
+    {
+        private int _length = 50;
+        private int _breadth = 20;
```

Comment:
These should be public properties instead of public fields like this :

```
        public int Breadth { get; set;}
        public int Length { get;  set;}
```